src/NServiceBus.Msmq.AcceptanceTests/Distributor/When_worker_processes_a_message.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace NServiceBus.AcceptanceTests.Distributor
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using NUnit.Framework;
+
+    public class When_worker_processes_a_message : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_also_send_a_ready_message()
+        {
+            var context = await Scenario.Define<DistributorEndpointTemplate.DistributorContext>()
```

Comment:
@danielmarbach we made the ScenarioContext no longer abstract because we don't want to enforce the definition of an "empty" context. Not sure why we would exactly do the opposite here?